src/Rest/Routes/NewsRoute.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * The class register route for $className endpoint
+ *
+ * @package Unicorns\Rest\Routes
+ */
+
+declare(strict_types=1);
+
+namespace Unicorns\Rest\Routes;
+
+use Unicorns\Config\Config;
+use UnicornsVendor\EightshiftLibs\Helpers\Components;
+use UnicornsVendor\EightshiftLibs\Rest\Routes\AbstractRoute;
+use UnicornsVendor\EightshiftLibs\Rest\CallableRouteInterface;
+
+/**
+ * Class NewsRoute
+ */
+class NewsRoute extends AbstractRoute implements CallableRouteInterface
+{
+
+	/**
+	 * Method that returns project Route namespace.
+	 *
+	 * @return string Project namespace UnicornsVendor\for REST route.
+	 */
+	protected function getNamespace(): string
+	{
+		return Config::getProjectRoutesNamespace();
```

Comment:
as I see project name space is wrong in you config It says it is Eightshift-libs and it should be your project namespace `unicorn`

Reply:
Should this be auto-populated with the project name when running the boilerplate setup?

Reply:
yes it should but that doesn't mean that you are free from not checking if everything is ok with you project, right?

Reply:
Of course I'm not free from checking :)

I was going in the direction that maybe I should check it out in the eightshift libs and see if there's a bug :D